docs/xrpl-wallet/api-reference/topup.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Topup
+---
+
+Top-up your cryptocurrency through our fiat-to-crypto partners with ease.
+
+---
+
+## `initiateTopup`
+
+Use the `initiateTopup` function to start a top-up transaction with the selected provider from the dapp. Check the list below for supported values of parameters and providers.
+
+:::info NOTE
+
+You don't need to be logged in with Torus to use the top-up API.
```

Comment:
is it correct? Does it work without login?

Reply:
Need to test, for now, removed this line.